spec/features/items/index_spec.rb

```diff
@@ -0,0 +1,25 @@
+require "rails_helper"
+
+describe "items index page" do
+  it "visitor can see all items" do
+    merchant_1 = Merchant.create(name: "Cecilia Chapman", address: "711-2880 Nulla St.", city: "Mankato", state: "MS", zip: 96522)
+    item_1 = merchant_1.items.create(
```

Comment:
Since this is the item index page where expect to see all items, I would make multiple items in your setup and expect them all to be on this page.